src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -26,15 +26,15 @@ public class ParserUtilTest {
     private static final String INVALID_EMAIL = "example.com";
     private static final String INVALID_GENDER = "A";
     private static final String INVALID_AGE = "A";
-    private static final String INVALID_TAG = "#friend";
+    private static final String INVALID_STUDY_GROUP_TAG = "#1A";
 
     private static final String VALID_NAME = "Rachel Walker";
     private static final String VALID_EMAIL = "rachel@example.com";
     private static final String VALID_GENDER = "F";
     private static final String VALID_AGE = "30";
     private static final String VALID_DETAIL = "detail";
-    private static final String VALID_TAG_1 = "friend";
-    private static final String VALID_TAG_2 = "neighbour";
+    private static final String VALID_STUDY_GROUP_TAG_1 = "friend";
```

Comment:
Perhaps can change the study_group_tag to 1A, 1B etc here as well?

Reply:
Thanks for noticing! I'll tweak it and merge afterwards~